LICENSE

```diff
@@ -1,5 +1,7 @@
 MIT License
 
+# test 1
+
```

Comment:
Thiago this code is awesome and elegant

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Efficient, Elegant, Reusable